pages/concepts/presence.mdx

```diff
@@ -20,48 +20,54 @@ r.subscribeToPresence((presentClientIDs) => {
   `presentClientIDs` will always contain the client's own client ID.
 </Callout>
 
-## Per-Client Presence State
+## Presence State
 
-The `subscribeToPresence` method only tells you the IDs of the clients that are present.
-To do something useful with them, you will combine these IDs with additional synced data about clients.
+The `subscribeToPresence` method only tells you the client IDs of the clients that are present.
```

Comment:
Nit but i feel like it was fine as-is.

Reply:
I wanted to be more explicit and distinguish between id and client id